app/src/main/res/layout/view_contributions_header.xml

```diff
@@ -1,36 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    tools:layout_width="match_parent"
-    tools:layout_height="wrap_content">
+    tools:layout_height="wrap_content"
+    tools:layout_width="match_parent">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="16dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginBottom="16dp"
-        android:gravity="center_vertical"
-        android:orientation="vertical">
+        android:layout_margin="16dp"
+        android:orientation="horizontal">
 
-        <TextView
-            android:id="@+id/contributionsCountText"
-            android:layout_width="match_parent"
+        <LinearLayout
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:lineSpacingExtra="8sp"
-            android:textColor="?attr/primary_text_color"
-            android:textSize="24sp"
-            android:textStyle="bold" />
+            android:layout_marginBottom="16dp"
+            android:layout_weight="1"
+            android:gravity="center_vertical"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/contributionsCountText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:lineSpacingExtra="8sp"
+                android:textColor="?attr/primary_text_color"
+                android:textSize="24sp"
+                android:textStyle="bold" />
+
+            <TextView
+                android:id="@+id/contributionsSeenText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="4dp"
+                android:lineSpacingExtra="8sp"
+                android:textColor="?attr/secondary_text_color"
+                android:textSize="16sp"
+                android:textStyle="normal" />
+        </LinearLayout>
 
-        <TextView
-            android:id="@+id/contributionsSeenText"
-            android:layout_width="match_parent"
+        <ImageView
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="4dp"
-            android:lineSpacingExtra="8sp"
-            android:textColor="?attr/secondary_text_color"
-            android:textSize="16sp"
-            android:textStyle="normal" />
+            android:contentDescription="@null"
+            app:srcCompat="@drawable/ic_edits_graph" />
```

Comment:
The margin to the left should be `16dp`, otherwise it will too close to the text.